src/components/vmd-appointment-card.component.ts

```diff
@@ -45,11 +61,16 @@ export class VmdAppointmentCardComponent extends LitElement {
                                 <div slot="content">
                                   <span class="fw-bold text-dark">${this.lieu.nom}</span>
                                   <br/>
-                                  <em>${this.lieu.metadata.address}</em>
+                                  <em>${this.renderAddress}</em>
```

Comment:
C'est peut-être mon coté old school, mais je suis pas ultra fan d'utiliser une property quand elle a verbe dans son nom

Ici, j'aurais plutôt appelé un `this.renderAddress()` en transformant la property en simple méthode (privée)

Ça reste très subjectif comme remarque, ça ne bloque en rien la PR

Reply:
Je partage ton avis, c'est corrigé. :slightly_smiling_face: